requirements.txt

```diff
@@ -0,0 +1,25 @@
+alembic==1.4.3
```

Comment:
@peterghrong I think this file should be in the backend folder instead of the root directory.

Reply:
You right legend, fixed !